samples/CloudClipboard/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,9 @@
+# Microsoft Open Source Code of Conduct
```

Comment:
This is something that would be a repository level asset; it should not exist for this sample alone.

Reply:
Removed